README_android.txt

```diff
@@ -19,26 +18,17 @@ Install the SDK
 ===============
 
 
-Start by extracting the SDK. You then need to add the SDK to your PATH
-environment variable. If you installed the SDK in $HOME/android-sdk, add
-$HOME/android-sdk/tools to your PATH.
-
-Once extracted you must run "android" (or android.bat on Windows) and install
-a version of the SDK. Allegro needs version 13 (3.1) of the SDK or higher to
-build. It can run on version 9 and up still though, if you include the
-correct uses-sdk fields in your manifest. See:
-https://developer.android.com/guide/topics/manifest/uses-sdk-element.html
-
-Also make sure the JAVA_HOME environment variable is set to your JDK
-directory (not always the case.)
+The most simple way is to install Android Studio which by default will
+place a copy of the SDK into ~/Android/Sdk.
 
 
 Install the NDK
 ===============
 
 
-Extract the NDK and add it to your PATH. If you installed the NDK in
-$HOME/android-ndk, add $HOME/android-ndk to your PATH.
+The best way is again to use Android Studio. Create a new project with
+C++ support and it will ask you if you want to install the NDK and will
+then proceed to place it into ~/Android/ndk-bundle.
```

Comment:
It's just a matter of downloading and extracting the archive... Thanks to previous instructions, I've easily and successfully managed to build Allegro for Android and integrate my cmake-based build system to automatically make APKs of my games, without installing the huge Android Studio at all. I don't feel like this is much of an upgrade :(

Reply:
The normal SDK and NDK also work! Also note that this PR is not finished and the current instructions will not work at all.

Reply:
Sure it works, but discovering how to make it work might be harder :) Just adding my two cents, as I was recently going through all the hassle of adding Android support to my Allegro based engine.

Reply:
The normal Allegro build builds an APK for each example and demo, without any Android Studio. I will add a note that you could just copy one of those as a template instead.